docs/content/integrations/import.md

```diff
@@ -320,6 +320,11 @@ Immuniweb Scan
 
 XML Scan Result File from Immuniweb Scan.
 
+IntSights Report
+--------------
+
+JSON report from IntSights ETP Suite API
+
```

Comment:
Could you add more details about where these reports come from? URL of the tools?
Specification?

Reply:
Certainly!